src/Numerics/LinearAlgebra/Complex/Matrix.cs

```diff
@@ -459,6 +459,75 @@ protected override void DoPointwiseLog(Matrix<Complex> result)
             Map(Complex.Log, result, Zeros.Include);
         }
 
+        protected override void DoPointwiseAbs(Matrix<Complex> result)
+        {
+            Map(x => (Complex)Complex.Abs(x), result, Zeros.Include);
```

Comment:
Ah, got it, thanks!  Will fix it for abs, acos, asin, atan, ceiling, floor, round, sin, sinh, sqrt, tan and tanh